crates/polars-parquet/src/arrow/write/boolean/basic.rs

```diff
@@ -79,12 +80,35 @@ pub fn array_to_page(
     )
 }
 
-pub(super) fn build_statistics(array: &BooleanArray) -> ParquetStatistics {
+fn distinct_bools(array: &BooleanArray) -> i64 {
+    let mut seen = 0b00i64;
```

Comment:
Boolean arrays have a bitmap with a cached null_count. We should at least use that one as a fast path when there are no nulls.